CHANGES.rst

```diff
@@ -29,6 +29,12 @@ Translations:
 Build:
  -
 
+Changes in Riot 0.8.18 (2018-10-13)
```

Comment:
Actually version 0.8.18 will be the next version. When submitting a PR you just have to add one line in the appropriate section of this file